README.md

```diff
@@ -4,14 +4,19 @@ A lightweight Python module to control the [SuperCollider](https://supercollider
 
 ## Installation
 
+##### v0.0.6 and higher
```

Comment:
No need to keep the old docs - liblo is now deprecated, so feel free to replace the instructions (and no need to mention liblo at all).

Reply:
I would still suggest to keep some information about this change in the README file. I made a second version: 861e77fe1a29c27d475489072747d3bb678bca19 using fewer lines. What do you think?

Reply:
Solved in [56e72bb](https://github.com/ideoforms/python-supercollider/pull/12/commits/56e72bb9b7ec30ab1c07e69eb9605b02ef0d49fb)